sql/src/test/java/io/crate/integrationtests/BlobTableIntegrationTest.java

```diff
@@ -52,4 +53,21 @@ public void testJoinWithBlobTable() throws Exception {
                 containsString("62cdb7020ff920e5aa642c3d4066950dd1f01f4d\n"),
                 containsString("bbe960a25ea311d21d40669e93df2003ba9b90a2\n")));
     }
+
+    @Test
+    public void testSortOnBlobTableColumn() throws Exception {
+        execute("create blob table b1 with (number_of_replicas = 0)");
+        ensureYellow();
+
+        for (String content : new String[]{"bar", "foo", "baz"}) {
+            upload("b1", content);
+        }
+        refresh();
+
+        execute("select b1.digest from blob.b1 order by b1.digest desc");
+        assertThat(TestingHelpers.printedTable(response.rows()),
+            is("bbe960a25ea311d21d40669e93df2003ba9b90a2\n" +
+               "62cdb7020ff920e5aa642c3d4066950dd1f01f4d\n" +
+               "0beec7b5ea3f0fdbc95d0dd47f3c5bc275da8a33\n"));
+    }
```

Comment:
Could we have a test with join and orderBy? (I don't know if this is even supported)